src/main/java/uk/gov/companieshouse/api/strikeoffobjections/config/ValidationConfig.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.companieshouse.api.strikeoffobjections.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.companieshouse.api.strikeoffobjections.common.ApiLogger;
+import uk.gov.companieshouse.api.strikeoffobjections.model.entity.ObjectionStatus;
+import uk.gov.companieshouse.api.strikeoffobjections.validation.AllowedValuesValidationRule;
+import uk.gov.companieshouse.api.strikeoffobjections.validation.DisallowedValuesValidationRule;
+import uk.gov.companieshouse.api.strikeoffobjections.validation.ValidationRule;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Configuration
+public class ValidationConfig {
+
+    @Value("#{'${ACTION_CODES_COMPANY_STRUCK_OFF}'.split(',')}")
+    private List<Long> companyStruckOffActionCodes;
+
+    @Value("#{'${ACTION_CODES_STRIKE_OFF_NOTICE}'.split(',')}")
+    private List<Long> strikeOffNoticeActionCodes;
+
+    @Bean
+    public List<ValidationRule<Long>> getActionCodeValidationRules(ApiLogger apiLogger) {
```

Comment:
I think we need to log the action codes that are read from config and being assigned to the various rules at some point - ideally only once and not every time that rules are run. Maybe that's in this method somewhere?

Reply:
i've got the rules logging on a failure but only set to debug. I could up it to info. I'll also look at logging the sets of numbers only once, probably happen on startup when the spring beans get autowired.

Reply:
I've added a private method to log out the values that is called when the getActionCodeValidationRules Bean is initialised on startup. I looked at having a constructor in this config class but the ApiLogger bean wasn't available to be injected into the constructor at that point during initialisation.

Reply:
just occurred to me I might be able to annotate a method with PostConstruct that will then have access to the the autowired beans (ApiLogger). I'll give that at try.

Reply:
Got it working with a @PostConstruct method in the config class. PR updated.